### DISCUSSION TITLE
Limit setzen: Frequenz

### DISCUSSION THREAD

**Discussion by Rustimation** [tbnobody/OpenDTU#1566] [Q&A]
Hallo Zusammen,
ich experimentiere gerade mt einem übrig gebliebenen HM-600 um eine Batterieeinspeisung für die produktionslose Zeit (v.a. nachts) zu realisieren. Das Einspeise-Limit (temporär) soll dynamisch an den Netzbezug angepasst werden um so möglichst nahe an Null heranzukommen. Das Ganze läuft mit Node-Red. Frage: Gibt es Erfahrungswerte, wie schnell hintereinander ich das Limit mit MQTT setzen kann? Bei ungebremster Regelung scheint der Input Buffer schnell vollzulaufen.
Wäre es eventuell besser, das über die API anstatt MQTT zu steuern?
Und eine letzte Frage: Was bedeutet die MeldungsID #2 "Time Calibration"?
Vielen Dank - gerne teile ich auch meine Erkenntnisse und Flows, wenn ich fertig bin.
Chris

**Comment by Marc9485** [comment]
#2 Time calibration
Bekomme ich bei meinen beiden Wr's jeden Tag angezeigt. Mal nur der eine, dann der andere und dann alle beide. Aber wenn beide dann nie gleichzeitig🤷🏻‍♂️ Also kanns aber nicht an der "Umgebung" liegen den von beiden die Panels haben die gleiche Ausrichtung und Winkel und keine verschattung. Tritt bei mir dann so 30-60 Minuten auf und dann ist Ruhe... Was das wörtlich bedeutet ist ja klar aber den Zusammenhang verstehe ich nicht...

**Reply by stefan123t** [reply]
Die Details zur AlarmEvent ID 2 "Time calibration" (früher "DTU command failed") habe ich hier bereits beschrieben: https://github.com/tbnobody/OpenDTU/discussions/1635#discussioncomment-8068107

**Comment by Marc9485** [comment]
Ich Probier das die Tage mal aus

https://github.com/tbnobody/OpenDTU/issues/1105

**Comment by Marc9485** [comment]
Netzteil mit mehr Leistung dran gemacht und den Kondensator. Keine Fehlermeldung mehr und die erreichbarkeit der Wr's ist zuverlässiger
